pagopa/io-app
# chore: [IOBP-363] Add ID Pay reward type info in initiative details screen

## Short description
This PR adds the absolute reward type info in the initiative details screens.

| Absolute | Percentage |
| --- | --- |
| <img src="https://github.com/pagopa/io-app/assets/6160324/27537cb0-f236-4c41-8a31-952b9f084164" width="200" /> | <img src="https://github.com/pagopa/io-app/assets/6160324/1ac3b53e-78bf-4f29-9770-dcb2cbd4155a" width="200" /> |

## List of changes proposed in this pull request
- Added required locale key
- Added reward rule logic to render correct row based on the `rewardValueType`

## How to test
With the `io-dev-server`, navigate to the ID Pay initiative details screen and check if the reward value row at the bottom of the screen is displayed correctly.
